cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -913,6 +909,17 @@ func (s *Supervisor) composeMergedConfig(config *protobufs.AgentRemoteConfig) (c
 				return false, fmt.Errorf("cannot merge config named %s: %w", name, err)
 			}
 		}
+	} else {
```

Comment:
Not sure if it should be in the readme or the design doc, but we should probably mention somewhere that the Collector will be run regardless of connection to an OpAMP server or the presence of backup local config to run.